src/app/component/dashboard/custom-grid/custom-grid.component.ts

```diff
@@ -60,7 +60,24 @@ export class CustomGridComponent implements OnInit {
   public method: any;
   public index: number;
   public dialogDelete = false;
-
+  public hiddenColumns: string[] = [];
+  public restoreColumns(): void {
+    this.hiddenColumns = [];
+  }
+  public hideColumn(field: string): void {
+    this.hiddenColumns.push(field);
+    console.log(this.hiddenColumns.toString())
+    let gridStorage = localStorage.getItem('kendo-grid-options')
```

Comment:
Nije primer dobre prakse.
Recimo, ukoliko kasnije zelis da menjas ime samog localStorage-a, moraces to da uradis na vise razlicitih mesta, da ne govorimo o tome da mozda mozes i da pogresis u nazivu.
Trebalo bi da se kreira get i set metoda u servisima i da se ovde samo pozove.
Pogledaj kako je to uradjeno u storage.service.ts.